code-examples/core_features/dev_setup.ts

```diff
@@ -0,0 +1,36 @@
+import * as kilt from '@kiltprotocol/sdk-js'
+import { Keyring } from '@kiltprotocol/utils'
+
+export async function main() {
+  // connect to the blockchain
+  await kilt.init({
+    address: 'wss://peregrine.kilt.io/parachain-public-ws'
```

Comment:
Shouldn't the dev setup rather enable a user to interact with a local blockchain instead of Peregrine? What's the purpose of listing zero balances of dev accounts? Do we expect the user to understand they can swap the endpoint for the local setup and print non-zero balances for the same accounts?

Reply:
Sorry. If I knew the PR was still WIP, I wouldn't have opened my mouth. Please continue to follow your plan and then I can give my 2 cents.